src/utils.ts

```diff
@@ -44,26 +44,6 @@ export function ellipsis( str: string, truncateLen: number, ellipsisChars?: stri
 }
 
 
-/**
- * Supports `Array.prototype.indexOf()` functionality for old IE (IE8 and below).
- *
- * @param {Array} arr The array to find an element of.
- * @param {*} element The element to find in the array, and return the index of.
- * @return {Number} The index of the `element`, or -1 if it was not found.
- */
-export function indexOf<T>( arr: T[], element: T ) {
```

Comment:
I think we might have had this function for IE8 support :X

Reply:
Maybe. But it was in error because of line 55. It says that it is always true because indexOf is always defined. Not sure why this error is now there. That's why I wasn't sure about doing this change.

Is it because now the recent versions of TypeScript don't support IE8 anymore? Or maybe this function is always defined in TypeScript even if IE8 it's not defined?

Reply:
Ahh, I see why we got that TypeScript error now. It's because TypeScript's "lib" config in `tsconfig.json` is set to "es2015", which makes TypeScript assume that we're building to an ES2015 JavaScript environment where certain functions will already exist (like `Array.prototype.indexOf`). Hence, as far as TypeScript is concerned, this `if` statement would always evaluate to `true`. 

IE8 only supports ES3 though which doesn't have the `Array.prototype.indexOf` method. And unfortunately, the lowest we can set that "lib" config to in `tsconfig.json` is "ES5" because TypeScript is too new! So we may just need a `// @ts-ignore` comment above that `if` statement